lib/news/widgets/article_screen.dart

```diff
@@ -34,8 +36,7 @@ class ArticleScreen extends StatelessWidget {
                 create: (_) =>
                     ArticleTheme(darkColor: Colors.purple, padding: padding),
                 child: ListView(
-                  padding: MediaQuery.of(context).padding +
-                      EdgeInsets.symmetric(horizontal: margin.toDouble()) +
+                  padding: EdgeInsets.symmetric(horizontal: margin.toDouble()) +
```

Comment:
Rather than converting margin `toDouble` here, we should probably do `width < 500 ? 0.0 : width * 0.08` above.
Also, we can simply use `EdgeInsets.symmetric(horizontal: ..., vertical: 16)` instead of `EdgeInsets.symmetric(horizontal: ...) + EdgeInsets.symmetric(vertical: 16)`